plugins/scala_test/main.py

```diff
@@ -0,0 +1,20 @@
+import argparse
+from update_imports import update_imports
+
+def _parse_args():
+    parser = argparse.ArgumentParser(description="Migrates scala tests!!!")
```

Comment:
Migrates them to what or from what? Also, longer term, do we need a parameter for a target version which affects which mappings we use?